ui/shared/components/panel/variants/selectors.js

```diff
@@ -0,0 +1,208 @@
+import { createSelector } from 'reselect'
+
+import { toSnakecase } from 'shared/utils/stringUtils'
+import {
+  NOTE_TAG_NAME,
+  EXCLUDED_TAG_NAME,
+  REVIEW_TAG_NAME,
+  KNOWN_GENE_FOR_PHENOTYPE_TAG_NAME,
+  DISCOVERY_CATEGORY_NAME,
+  SORT_BY_FAMILY_GUID,
+  VARIANT_SORT_LOOKUP,
+  SHOW_ALL,
+  VARIANT_EXPORT_DATA,
+} from 'shared/utils/constants'
+import {
+  getVariantTagsByGuid, getVariantNotesByGuid, getTagsByVariantGuids, getNotesByVariantGuids, getSavedVariantsByGuid,
+  getFamiliesByGuid, getAnalysisGroupsByGuid, getGenesById, getUser,
+} from 'redux/selectors'
+
+
+// Saved variant selectors
+export const getSavedVariantTableState = state => (
+  state.currentProjectGuid ? state.savedVariantTableState : state.allProjectSavedVariantTableState
+)
+
+const matchingVariants = (variants, matchFunc) =>
+  variants.filter(o => (Array.isArray(o) ? o : [o]).some(matchFunc))
+
+export const getPairedSelectedSavedVariants = createSelector(
+  getSavedVariantsByGuid,
+  (state, props) => props.match.params,
+  getFamiliesByGuid,
+  getAnalysisGroupsByGuid,
+  (state, props) => (props.project || {}).projectGuid,
```

Comment:
this is the only thing that really changed in this PR - this used to come from the `getCurrentProject` selector which should only ever be used for project page specific components and selectors, so I changed this to instead have a project prop passed explicitly if it is supposed to be defined